src/Indice.Features.Cases.App/src/app/features/cases/cases.base.html

```diff
@@ -0,0 +1,46 @@
+<lib-view-layout title="Υποθέσεις"
+                 [busy]="!items"
+                 [actions]="formActions"
+                 [meta-items]="metaItems"
+                 (onAction)="actionHandler($event)"
+                 (onSearch)="searchChanged($event)">
+    <div class="text-right">
+        <button class="btn-submit"
+                [ngClass]="{'opacity-50 cursor-not-allowed': !queryParamsHasFilter}"
+                [disabled]="!queryParamsHasFilter"
+                (click)="openQueryModal()">
+            Αποθήκευση Αναζήτησης</button>
+    </div>
+    <lib-list-view [items]="items"
+                   [search-options]="searchOptions"
```

Comment:
Αν καταλαβαίνω απο το code duplication που βλέπω στο πως φτιαχνονται τα search options, έχεις πρόβλημα γιατί πρέπει να βάλεις κώδικα μέσα στο subscribe ώστε ΌΤΑΝ έρθουν τα δεδομένα να κάνεις μαγίες. Το πρόβλημα θα σου λυθεί αν ΔΕΝ κανεις subscribe και τα searchOptions μείνουν σαν Observable<SearchOptions>. Έτσι μπορείς να έχεις μια base υλοποίηση που γυρίζει Observable<SearchOptions>. To menu item component μπορεί μετα να καλέσει την base, να πάρει το Observable<SearchOptions> και να του κάνει .pipe(φερε και τα menu items να προσθέσω φιλτρα).

Τέλος θα έχεις ενα objervable<searchoption> και οχι σκετο searchOptions. ΑΥτό λύνεται αν κανεις subscribe στο view με [search-options]="searchOptions | async"